Common/src/geometry_structure.cpp

```diff
@@ -6792,7 +6792,7 @@ void CPhysicalGeometry::PrepareOffsets(unsigned long val_npoint_global) {
   nPoint_Cum = new unsigned long[size+1];
   
   unsigned long quotient = val_npoint_global/size;
-  unsigned long remainder = val_npoint_global%size;
+  int remainder = int(val_npoint_global%size);
   for (int ii = 0; ii < size; ii++) {
     nPoint_Lin[ii] = quotient + int(ii < remainder);
```

Comment:
Cast the comparison to `unsigned long` instead of changing the type of remainder, I wrote `int` because I'm lazy.

Reply:
The comparison raised a warning because ii is an int and remainder was an unsigned long.
Now it does not complain but I understand that int(ii<remainder) should be changed to
(unsigned long)(ii<remainder). Correct ?

Reply:
Ah yes... Never mind that then.